dvc/exceptions.py

```diff
@@ -286,3 +286,10 @@ def __init__(self, url):
         super(UrlNotDvcRepoError, self).__init__(
             "URL '{}' is not a dvc repository.".format(url)
         )
+
+
+class GetDVCFileError(DvcException):
+    def __init__(self):
+        super(GetDVCFileError, self).__init__(
+            "the given path is a DVC-file, you must specify an output"
```

Comment:
- can we print the `given path` to make it easier to understand what path do we mean

- decipher the output term to make it user friendly

Reply:
@shcheklein , it is already printed in the beginning of the error message:

```bash
ERROR: failed to get 'scripts/innosetup/dvc.ico.dvc' from 'https://github.com/iterative/dvc' - the given path is a DVC-file, you must specify an output
```

What do you suggest instead of output? _data artifact_ ?

Reply:
why don't `specify a data file or directory path` or something like this?

Reply:
done, @shcheklein